script.js

```diff
@@ -0,0 +1,216 @@
+    return 'YOU LOSE';
+  }
+
+  return 'odd call 😬';
+};
+
+const playRoundForRock = () => {
+  const player = gameBodyRockTextPlayer.textContent;
+  const computer = gameBodyRockTextComputer.textContent;
+
+  if (player === 'rock' && computer === 'rock') {
+    changeComRock();
+    return 'DRAW';
+  }
+
+  if (player === 'rock' && computer === 'scissors') {
+    changeComScissors();
+    playerScore.textContent = `${(playerGoal += 1)}`;
+    return 'YOU WIN';
+  }
+  if (player === 'rock' && computer === 'paper') {
+    changeComPaper();
+    computerScore.textContent = `${(computerGoal += 1)}`;
+    return 'YOU LOSE';
+  }
+
+  return 'odd call 😬';
+};
+
+const playRoundForScissors = () => {
+  const player = gameBodyScissorsTextPlayer.textContent;
+  const computer = gameBodyScissorsTextComputer.textContent;
+
+  if (player === 'scissors' && computer === 'scissors') {
+    changeComScissors();
+    return 'DRAW';
+  }
+
+  if (player === 'scissors' && computer === 'paper') {
+    playerScore.textContent = `${(playerGoal += 1)}`;
+    changeComPaper();
+    return 'YOU WIN';
+  }
+  if (player === 'scissors' && computer === 'rock') {
+    changeComRock();
+    computerScore.textContent = `${(computerGoal += 1)}`;
+    return 'YOU LOSE';
+  }
+
+  return 'odd call 😬';
+};
+
+const getPlayerRock = () => {
+  gameBodyRockTextPlayer.textContent = 'rock';
+  gameBodyRockTextComputer.textContent = getComputerChoice();
+  const playerSvgRock = document.querySelector('.player-option-svg-player path');
+
+  playerSvgRock.setAttribute('d', dPathRock);
+  gameBodyBigCirclePlayerOptionBorder.setAttribute('class', 'change-rock-circle-border');
+
+  gameBodyDiv.style.display = 'none';
+  afterPlayerComputerChoiceDiv.style.display = 'block';
+  winLoseText.textContent = playRoundForRock();
+};
+
+const getPlayerScissors = () => {
+  gameBodyScissorsTextPlayer.textContent = 'scissors';
+  gameBodyScissorsTextComputer.textContent = getComputerChoice();
+  const playerSvgRock = document.querySelector('.player-option-svg-player path');
+
+  playerSvgRock.setAttribute('d', dPathScissors);
+  gameBodyBigCirclePlayerOptionBorder.setAttribute('class', 'change-scissors-circle-border');
+
+  gameBodyDiv.style.display = 'none';
+  afterPlayerComputerChoiceDiv.style.display = 'block';
+  winLoseText.textContent = playRoundForScissors();
+};
+
+const getPlayerPaper = () => {
+  gameBodyPaperTextPlayer.textContent = 'paper';
+  gameBodyPaperTextComputer.textContent = getComputerChoice();
+  const playerSvgRock = document.querySelector('.player-option-svg-player path');
+
+  playerSvgRock.setAttribute('d', dPathPaper);
+  gameBodyBigCirclePlayerOptionBorder.setAttribute('class', 'change-paper-circle-border');
+
+  gameBodyDiv.style.display = 'none';
+  afterPlayerComputerChoiceDiv.style.display = 'block';
+
+  winLoseText.textContent = playRoundForPaper();
+};
+
+const playAgain = () => {
+  gameBodyDiv.style.display = 'flex';
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList Take a look at this, you should not be making use of style to display because if you are doing such you are mixing javascript with CSS and vice versa, making use of classList will ensure that you make use of javascript for javascript aspect and not mix it with css